src/private/files.ts

```diff
@@ -487,11 +487,17 @@ export namespace files {
    * @private
    * Hide from docs
    *
-   * Open download preference folder
+   * Open download preference folder or folder containing filePath if its value is non-null
+   * @param filePath: Path of the file whose containing folder should be opened
```

Comment:
I'm not sure I understand why you are passing a path to a file into an "open a folder" function. Shouldn't you be passing in a folder path? The opening description is a little confusing:

"Open download preference folder or folder containing filePath if its value is non-null"

Does that mean that callers are intended to pass in a null filePath if they want to open the download preference folder? If that's the case, shouldn't filePath be an optional parameter?

Reply:
I agree with the fact that the callers should be given flexibility to not pass null for opening download preference folder. For now i have assigned default null value to filePath variable and in v2.0 will make sure to make it optional.